osmnames/export_osmnames/create_polygons_view.sql

```diff
@@ -27,6 +27,6 @@ SELECT
   wikipedia AS wikipedia
 FROM
   osm_polygon,
-  get_parent_info(osm_polygon.name, parent_id, place_rank) AS parentInfo,
+  get_parent_info(name, id, parent_id, True, type) as parentInfo,
```

Comment:
Instead of adding a new parameter `id` to the `get_parent_info` method, couldn't we just call the method with the id instead of the parent_id ?  `get_parent_info(name, id, True, type)`